.gitlab-ci-defaults.yml

```diff
@@ -0,0 +1,14 @@
+# SaaS customers may extend this if unlocked in Gitlab. Some types of
+# configuration may be restricted.
+
+# Template values for linting and unit tests, eg phpunit.
+.static:
```

Comment:
this name is way too confusing. `.static` usually refers to static assets. I would thing that this is used to compile static assets, like minify JS etc.

 Can this be called `lint` ?

Reply:
It includes any sort of static test, which includes phpunit static tests. Have considered renaming it. TBC.